<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjetoFinal</class>
 <widget class="QMainWindow" name="ProjetoFinal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjetoFinal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>video original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>video processado</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>341</y>
      <width>81</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="FuturePastLayout">
     <item>
      <widget class="QCheckBox" name="futureButtom">
       <property name="text">
        <string>Future</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="pastButtom">
       <property name="text">
        <string>Past</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="speedLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speedSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="closeButtom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encerrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runDemoKalman">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>72</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RunDemoKalman</string>
    </property>
   </widget>
   <widget class="QComboBox" name="videoFileOption">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>178</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="PauseRestartLayout">
     <item>
      <widget class="QPushButton" name="pauseButtom">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButtom">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearButtom">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="inputImg">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="outputImg">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>411</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="ColorMinLabel">
         <property name="text">
          <string>ColorMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ColorMinChannelOption"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="ColorMinValue">
           <property name="text">
            <string>value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ColorMinChannelSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="ColorMaxLabel">
         <property name="text">
          <string>ColorMax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ColorMaxChannelOption"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="ColorMaxValue">
           <property name="text">
            <string>value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ColorMaxChannelSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>252</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="DefineColorLabel">
       <property name="text">
        <string>DefinedColor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DefinedColorOption"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>450</y>
      <width>412</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="BackGroundSubButtom">
       <property name="text">
        <string>BackGroundSubtract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ColorDetectionButtom">
       <property name="text">
        <string>ColorDetection</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="BorrarButtom">
         <property name="text">
          <string>Borrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SizeOfGaussian"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="GlobalModeLabel">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>71</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="GlobalModeOption">
    <property name="geometry">
     <rect>
      <x>956</x>
      <y>71</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearPastButtom">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Past</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ColorDetectionThresholdLabel">
       <property name="text">
        <string>ColorDetectionTh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ColorDetectionThresholdSliderValue">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ColorDetectionThresholdSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>logText</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>closeButtom</zorder>
   <zorder>runDemoKalman</zorder>
   <zorder>videoFileOption</zorder>
   <zorder>clearButtom</zorder>
   <zorder>GlobalModeLabel</zorder>
   <zorder>GlobalModeOption</zorder>
   <zorder>ClearPastButtom</zorder>
   <zorder>ColorDetectionThresholdLabel</zorder>
   <zorder>ColorDetectionThresholdSlider</zorder>
   <zorder>ColorDetectionThresholdSliderValue</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
